customer-address.go

```diff
@@ -18,6 +20,41 @@ func newCustomerAddressRepository(client http.Client, createURL func(endpoint st
 	}
 }
 
+func (r customerAddressRepository) Create(id int64, address shopify.CustomerAddress) (shopify.CustomerAddress, error) {
+	url := r.createURL(fmt.Sprintf("customers/%v/addresses.json", id))
+
+	addressDTO := BuildCustomerAddressDTO(address)
+
+	// Wrapping Address object inside `address` property
+	request := struct {
+		Address CustomerAddressDTO `json:"address"`
+	}{
+		Address: addressDTO,
+	}
```

Comment:
What you think of this?

```suggestion
	request := struct {
		CustomerAddressDTO `json:"address"`
	}{
		addressDTO,
	}
```